db/migrate/20190916105800_create_zipcode_rates_archives.rb

```diff
@@ -0,0 +1,15 @@
+class CreateZipcodeRatesArchives < ActiveRecord::Migration
+  def up
+    safety_assured do
```

Comment:
I'm a little late - j/w what `strong_migrations` was complaining about that made you wrap with `saftey_assured`?

Reply:
`strong_migrations` doesn't inspect direct SQL statements so will flag it as a "Possibly dangerous operation" and tell you to think carefully about your life choices before wrapping it w/ `safety_assured`